include/swift/AST/Types.h

```diff
@@ -601,6 +601,9 @@ class alignas(1 << TypeAlignInBits) TypeBase
   /// Is this the 'Any' type?
   bool isAny();
 
+  /// Is this the 'any Sendable' type?
+  bool isAnySendable();
```

Comment:
Can we generalize this to be "`Any` ignoring marker protocols"?

Reply:
Alright, fine.